### ISSUE TITLE
[Bug] YTVanced force closes on startup 

### ISSUE THREAD

**Issue by dbpm1** [YTVanced/Vanced#623]
**Bug description**
After updating to this latest version, the YTVanced force closes only after restarting the phone.

**EDIT** Since I didn't checked which previous version I had before updating, I tested version 15.43.32 and the issue remains, only on version 15.40.37 the problem is solved. 

Cannot open even after reinstalling it in manager, one possible workaround is a complete uninstall and reinstall but after another hot or cold restart of the OS the issue remains.
If not restarted, the app will function without a problem even for days of uptime. 
Possible duplicate of  #615. 
There is this summary of the error provided by the OS:

```
java.lang.RuntimeException: Unable to get provider com.google.android.apps.youtube.app.common.util.MainAppFileProviderCompat: java.lang.IllegalArgumentException: Missing android.support.FILE_PROVIDER_PATHS meta-data
	at android.app.ActivityThread.installProvider(Unknown Source:429)
	at android.app.ActivityThread.installContentProviders(Unknown Source:28)
	at android.app.ActivityThread.handleBindApplication(Unknown Source:1227)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.swift.sandhook.SandHook.callOriginMethod(SandHook.java:183)
	at com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge(HookStubManager.java:380)
	at SandHookerNew_42b8453bf7df105343083f1377c32094be493c49.hook(Unknown Source:51)
	at android.app.ActivityThread.access$1400(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(Unknown Source:710)
	at android.os.Handler.dispatchMessage(Unknown Source:19)
	at android.os.Looper.loop(Unknown Source:242)
	at android.app.ActivityThread.main(Unknown Source:98)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(Unknown Source:11)
	at com.android.internal.os.ZygoteInit.main(Unknown Source:275)
Caused by: java.lang.IllegalArgumentException: Missing android.support.FILE_PROVIDER_PATHS meta-data
	at hn.b(PG:40)
	at hn.attachInfo(PG:6)
	at acei.attachInfo(PG:10)
	at android.app.ActivityThread.installProvider(Unknown Source:246)
	... 14 more

```

![IMG-20210405-WA0022](https://user-images.githubusercontent.com/82009615/113652542-56174f00-966a-11eb-9c9d-7e3916748bba.jpg)

![IMG-20210405-WA0023](https://user-images.githubusercontent.com/82009615/113652564-5fa0b700-966a-11eb-8ae3-c59d8e74730d.jpg)

**Variant**
Root

**Vanced version**
16.02.35

**Android version**
Android 10 - MIUI 11.2 by xiaomi.eu 20.3.12

**Device**
Xiaomi Mi 9 

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
2. Force close 

**Expected behavior**
Since the error appears to be related to this version of the YTVanced, I would expect that it should function like the previous one. 
**EDIT** version 15.40.37 do not force closes after restart.

 Would like to add that I have not updated or modified settings in any other app or module related to the root access (Magisk/EdXposed with SandHook/CustoMIUIzer).

**Comment by dbpm1** [comment]
Complementary information: I just read the "Vanced stuck loading / Music unable to login" guide and while the workaround is almost the same as the one I've done, I don't have either one of the microG packages installed.

**Comment by gghhkm** [comment]
Try this
1. Change Root Script Sleep Time (I recommend 2, but going higher than that can also work for some)
2. Uninstall Vanced
3. Use a Root browser to remove `data/adb/Vanced` and `data/adb/service.d/vanced.sh`
4. Reboot 
Youtube should be normal again
5. Install Vanced once more
6. Check if Vanced is working after rebooting.
7. Done.

**Comment by dbpm1** [comment]
Work like a charm, can confirm thatvversion 16.02.35 is a ok. Thanks for the quick reply @gghhkm!
